whole-body-controller-core/settings.gradle.kts

```diff
@@ -0,0 +1,13 @@
+buildscript {
+   repositories {
+      maven { url = uri("https://plugins.gradle.org/m2/") }
+      mavenLocal()
+   }
+   dependencies {
+      classpath("us.ihmc:ihmc-build:0.20.2")
+   }
+}
+
+val ihmcSettingsConfigurator = us.ihmc.build.IHMCSettingsConfigurator(settings, logger, extra)
+ihmcSettingsConfigurator.configureAsGroupOfProjects()
+ihmcSettingsConfigurator.findAndIncludeCompositeBuilds()
```

Comment:
This will also include `repository-group` as a dependency, which I don't believe is needed (please confirm though).

You can instead just do this:

```kotlin
buildscript {
   repositories {
      maven { url = uri("https://plugins.gradle.org/m2/") }
      mavenLocal()
   }
   dependencies {
      classpath("us.ihmc:ihmc-build:0.20.2")
   }
}

rootProject.name = "whole-body-controller-core"
includeBuild("mobile-simulation")
includeBuild("robot-arm-one")
includeBuild("robot-arm-two")
includeBuild("robot-arm-three")
includeBuild("robot-walker-four")
```

Reply:
Well this is using our Gradle build plugin, I don't really feel like messing with it.